src/Serilog.Sinks.AzureTableStorage/Sinks/KeyGenerator/DefaultKeyGenerator.cs

```diff
@@ -0,0 +1,27 @@
+using System.Threading;
+using Serilog.Events;
+
+namespace Serilog.Sinks.AzureTableStorage.KeyGenerator
+{
+    class DefaultKeyGenerator : IKeyGenerator
+    {
+        protected long RowId;
+
+        public DefaultKeyGenerator()
+        {
+            RowId = 0L;
+        }
+
+        public string GeneratePartitionKey(LogEvent logEvent)
+        {
+            var utcEventTime = logEvent.Timestamp.UtcDateTime;
+            var timeRoundedToMinute = utcEventTime.AddMilliseconds(-utcEventTime.Millisecond);
+            return $"0{timeRoundedToMinute.Ticks}";
+        }
+
+        public string GenerateRowKey(LogEvent logEvent)
+        {
+            return $"{Interlocked.Increment(ref RowId)}";
```

Comment:
The previous implementation had `string.Format("{0}|{1}", log.Level, log.MessageTemplate.Text)` - should we keep that to avoid a breaking change?

This implementation could also be provided as a separate class if it was felt it's useful?